### DISCUSSION TITLE
Internal chat feature

### DISCUSSION THREAD

**Discussion by iw4p** [chatwoot/chatwoot#9731] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

Hi, first of all thank you for internal chat fork. I am trying to use it but it has some simple bugs. It's essential for me to have chatwoot with internal chat. After running your code (I added them on top of the last commits of chatwoot) I realized it works fine but I don't know how to start a new conversation after creating a new inbox. Should not it create a new one automatically?

### Describe the solution you'd like

1. Be able to create inbox (done)
2.  Be able to add agents to that inbox (done)
3. Show an automated initiated conversation on that inbox (I need help here)

### Describe alternatives you've considered

I tried to create a new conversation but it needs email and source_id which must be for a contact, not an agent.
(I created a conversation using API and add a contact but there's an Echo error so when I write something, that contact will echo that message to the conversation again)

**Comment by pranavrajs** [comment]
@iw4p I'm not sure about the fork you are mentioning. Please provide more information here. We don't support internal chats. 

Moving this to discussions.

**Comment by iw4p** [comment]
Hi, I mean @clairton fork on [https://github.com/chatwoot/chatwoot/pull/7448](https://github.com/chatwoot/chatwoot/pull/7448) which is WIP.

**Reply by pranavrajs** [reply]
Sorry, we are not considering this as a feature right now since it is not the core problem we are solving.
